.github/workflows/release.yml

```diff
@@ -210,25 +210,24 @@ jobs:
           if [ "$TICKET_TYPE" == "Epic" ]; then
```

Comment:
1. The code changes made in the curl request URL have removed the 'self' field from the request parameters. This means that the response will no longer include the 'self' URL for the assignee.

2. The jq filter used to extract data from the Jira API response has been modified. The assignee field is now accessed using '.fields.assignee.name' instead of '.fields.assignee.displayName'. This will affect how the assignee name is extracted and displayed in the subsequent processing.

3. In the awk script, the variable 'assignee_url' has been removed as it is no longer part of the extracted data. The assignee field is now being processed differently to include the '@' symbol before the assignee name.

4. The printf statement in the awk script has been updated to reflect the changes in data extraction. The assignee field no longer includes the URL and is formatted differently. The issue link, status, and new assignee format are printed in the table.